docker/docker_client.go

```diff
@@ -423,7 +423,13 @@ func SearchRegistry(ctx context.Context, sys *types.SystemContext, registry, ima
 				res := SearchResult{
 					Name: repo,
 				}
-				searchRes = append(searchRes, res)
+				// bugzilla.redhat.com/show_bug.cgi?id=1976283
```

Comment:
(Absolutely non-blocking: I’m not sure the bug number is that relevant for the future; having it in the commit message and this PR might be enough. OTOH it doesn’t really hurt.)

Reply:
I find bug and issue numbers in code comments incredibly helpful when reading code.  Browsing the git history works but takes much longer.

Reply:
Bug numbers are useful if they contain a larger discussion about finding the root cause, or about trade-offs. In this case I’m not sure what it brings over the comment just below; it is mostly side-tracks.

Anyway, it doesn’t hurt.